test/reload.bats

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+    start_crio
+}
+
+function teardown() {
+    cleanup_test
+}
+
+function replace_config() {
+    sed -ie 's/\('$1' = "\).*\("\)/\1'$2'\2/' "$CRIO_CONFIG"
+}
+
+function reload_crio() {
+    # send SIGHUP
+    kill -HUP $CRIO_PID
+
+    # give it a bit time to deliver the signal
+    sleep 1
```

Comment:
The helpful reviewer might have a better approach here.

Reply:
We can store a reload attempt with success or failure status in the /info endpoint (server/inspect.go). We can then query the endpoint in a loop with a sleep to know whether the attempt succeeded or not and then proceed.

Reply:
Thanks for the hint :pray:, I will go for that.